parmed/topologyobjects.py

```diff
@@ -2539,34 +2537,41 @@ def from_rbtorsion(cls, rbtorsion):
 
         Raises
         ------
-        AttributeError if rbtorsion does not have the attributes c0, c1, c2, c3,
-        c4, and c5.
+        ValueError if all terms in rbtorsion are zero except for c0
 
-        ValueError if c5 is not 0 and c1+c2+c3+c4 != 0
         """
-        # c0 is ignored (it only contributes a constant...)
+        c0 = rbtorsion.c0
         c1 = rbtorsion.c1
         c2 = rbtorsion.c2
         c3 = rbtorsion.c3
         c4 = rbtorsion.c4
         c5 = rbtorsion.c5
-        if c5 != 0:
-            raise ValueError('Cannot convert R-B torsion with c5 != 0')
-        if abs(c1 + c2 + c3 + c4) > TINY:
-            raise ValueError('Cannot convert R-B torsion with c1+c2+c3+c4 != 0')
-        # These conversions are based on the gromacs manual (2016.1, eqn 4.65)
-        f1 = -2.0 * c1 - 3.0 * c3 / 2.0
-        f2 = -c2 - c4
-        f3 = -c3 / 2.0
-        f4 = -c4 / 4.0
+
+        phi = 0 * u.degrees
+        fc0 = 1.0*c0 + 0.5*c2 + 0.3750*c4
+        fc1 = 1.0*c1 + 0.75*c3 + 0.6250*c5
+        fc2 = 0.5*c2 + 0.5*c4
+        fc3 = 0.25*c3 + 0.3125*c5
+        fc4 = 0.125*c4
+        fc5 = 0.0625*c5
```

Comment:
I am using the same tests that @maaren implemented. This is the conversion to a dihedral of the form 
```
fc0 + sum_i=1^6 fci (cos(nx-phi)
```
which can be decomposed into proper dihedrals. I will add more tests tomorrow to make sure all the bases are covered and my validation issue is resolved but we have been using this as our canonical dihedral form in InterMol.

More info: https://github.com/shirtsgroup/InterMol/blob/master/intermol/gromacs/gromacs_parser.py#L205-L218